io/io/src/RRawFileUnix.cxx

```diff
@@ -108,6 +112,23 @@ void ROOT::Internal::RRawFileUnix::OpenImpl()
    }
 }
 
+void ROOT::Internal::RRawFileUnix::ReadVImpl(RIOVec *ioVec, unsigned int nReq)
+{
+#ifdef R__HAS_URING
+   if (!RIoUring::IsAvailable()) {
+      R__DEBUG_HERE("RRawFileUnix") <<
+         "io_uring setup failed, falling back to default ReadV implementation";
+      RRawFile::ReadVImpl(ioVec, nReq);
+      return;
+   }
+   // check we can construct the ring
+   RIoUring ring(8);
+   throw std::runtime_error("io_uring ReadV unimplemented!");
```

Comment:
Let's instead use `RRawFile::ReadVImpl(ioVec, nReq);` and add a `TODO` in order to not generate unexpected failures where liburing is available (RRawFile is already used in the sqlite RDataFrame data source)

Reply:
Done in 41858029cba8cc7c55d4bf9ae1440f8269fe091a